.github/workflows/fitness-functions.yml

```diff
@@ -17,12 +17,13 @@ jobs:
         with:
           node-version: '16'
 
+      - name: Install dependencies
+        run: yarn
+
       - name: Run fitness functions
         env:
-          HEAD_REF: ${{ github.event.pull_request.head.ref }}
           BASE_REF: ${{ github.event.pull_request.base.ref }}
         run: |
-          # git fetch origin $HEAD_REF
-          # git fetch origin $BASE_REF
-          # git diff origin/$BASE_REF origin/$HEAD_REF -- . > diff
-          # npm run fitness-functions -- "ci" "./diff"
+          git fetch origin $BASE_REF
```

Comment:
Nit: One alternative would be to set `depth` to `0` in the checkout step earlier in this workflow. The default depth is 1, but setting it to zero fetches the entire history. This is the conventional way to ensure there is sufficient git history for operations that reference older commits.